mqtt/mqtt-broker/src/auth/mod.rs

```diff
@@ -27,6 +27,15 @@ pub enum AuthId {
     Identity(Identity),
 }
 
+impl AuthId {
+    pub fn as_str(&self) -> &str {
+        match self {
+            AuthId::Anonymous => "*",
+            AuthId::Identity(identity) => identity.as_str(),
+        }
+    }
+}
+
 impl Display for AuthId {
     fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
         match self {
```

Comment:
Could you please change this method as well?
```suggestion
        write!(f, self.as_str())
```